mock/breez-sdk/src/lib.rs

```diff
@@ -1041,7 +1096,6 @@ impl BreezServices {
                 reverse_swap_info: None,
             });
 
-            *LN_BALANCE_MSAT.lock().unwrap() -= amount_msat;
```

Comment:
Yeah okay I see, it's not strictly necessary ... But why did you remove this? I thought it would keep the wallet coherent

Reply:
Do you mean the total balance variable? We now use channels as state. Having the variable introduced information duplication and forces us always to update both parts in all the places

Reply:
We clarified in a side-channel that @gcomte was referring to the mismatch between the balance and payments that is created by my changes to `simulate_payments()`. We nonetheless agreed the mismatch is not important so no changes are needed.